includes/active-directory-protocols-getting-started.md

```diff
@@ -26,7 +26,7 @@ First, you need to register your application with your Azure Active Directory (A
 * Follow the prompts and create a new application. It doesn't matter if it is a web application or a public client (mobile & desktop) application for this tutorial, but if you'd like specific examples for web applications or public client applications, check out our [quickstarts](../articles/active-directory/develop/v1-overview.md).
   * **Name** is the application name and describes your application to end users.
   * Under **Supported account types**, select **Accounts in any organizational directory and personal Microsoft accounts**.
-  * Provide the **Redirect URI**. For Web Applications, this is the base URL of your app where users can sign in.  For example, `http://localhost:12345`. For public client (mobile & desktop), Azure AD uses it to return token responses. Enter a value specific to your application.  For example, `http://MyFirstAADApp`.
+  * For Native Applications provide a **Redirect URI**, which Azure AD will use to return token responses. Enter a value with a custom scheme, .e.g `urn:ietf:wg:oauth:2.0:oob`
```

Comment:
@CelesteDG - sorry (my previous comment was deleted when i cancelled the review). In any case, please advise on this PR. I've recreated the change (overwritten during the merge conflict resolution). Do you want the new content (already in master), the content in this PR, or some combination of the two?

Thanks!